src/plugins/bfetch/server/plugin.ts

```diff
@@ -208,23 +208,15 @@ export class BfetchServerPlugin
     >(path, (request) => {
       const handlerInstance = handler(request);
       return {
-        getResponseStream: ({ batch }) => {
-          const subject = new Subject<BatchResponseItem<BatchItemResult, E>>();
-          let cnt = batch.length;
-          batch.forEach(async (batchItem, id) => {
+        getResponseStream: ({ batch }) =>
+          map$(batch, async (batchItem, id) => {
```

Comment:
The subtle difference here is that `map$()` won't actually trigger calls to `onBatchItem()` until the observable returned is subscribed, and will re-execute the requests every time the observable is re-subscribed. This removes what I think was a race condition, where results and errors could technically be emitted from the previous `Subject` before the caller subscribed. It also makes the code a little simpler. Hope you agree.